### PR TITLE
add memory cache options

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: tests/Microsoft.Identity.Web.Test/CacheExtensionsTests.cs [hunk 1]
```diff
@@ -53,6 +53,40 @@ public void InMemoryCacheExtensions_NoCca_ThrowsException_Tests()
...
+            }));
+
+            Assert.Equal("confidentialClientApp", ex.ParamName);
+        }
+
+        [Fact]
+        public void InMemoryCache_WithServices_NoService_ThrowsException_Tests()
+        {
+            CreateCca();
+
```
Comment (ciaozhang) [inline tests/Microsoft.Identity.Web.Test/CacheExtensionsTests.cs:84]: nit: extra line

Comment (jennyf19) [reply tests/Microsoft.Identity.Web.Test/CacheExtensionsTests.cs]: fixed.

**Review by ciaozhang — APPROVED** [review]
LGTM @jennyf19

File: src/Microsoft.Identity.Web.TokenCache/TokenCacheExtensions.cs [hunk 2]
```diff
@@ -110,7 +103,13 @@ public static class TokenCacheExtensions
         /// The following code adds an in-memory token cache.
         ///
         /// <code>
-        ///  app.AddInMemoryTokenCache();
+        ///  app.AddInMemoryTokenCache(services =>
+        ///  {
+        ///       services.Configure&lt;MsalMemoryTokenCacheOptions&gt;(options =>
+        ///       {
+        ///           options.AbsoluteExpirationRelativeToNow = TimeSpan.FromHours(1); // for example
+        ///       });
```
Comment (jmprieur) [inline src/Microsoft.Identity.Web.TokenCache/TokenCacheExtensions.cs:111]:
```suggestion
        ///    // Configure the memory cache options
        ///    services.Configure&lt;MemoryCacheOptions&gt;(options =>
        ///    {
        ///       options.SizeLimit = 5000000; // in bytes (5 Mb)
        ///    });
        ///    }
 ```

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @jennyf19 

i used the MemoryCacheOptions directly here as this is more fine grain, and this offers way more control

File: src/Microsoft.Identity.Web.TokenCache/TokenCacheExtensions.cs (hunk 2 above)
Comment (jennyf19) [reply src/Microsoft.Identity.Web.TokenCache/TokenCacheExtensions.cs]: got it. fixed.

**Pull request merged**